frontend-mobile/lib/view/settings_screen.dart

```diff
@@ -9,9 +9,6 @@ class SettingsScreen extends StatelessWidget {
         appBar: AppBar(
           title: const Text('Settings'),
         ),
-        body: const Center(
-            child: Text('Hello Settings')
-        )
-    );
+        body: const Center(child: Text('Hello Settings')));
```

Comment:
If you want to keep indentation, add a comma between the braces. It's annoying, I know.

Reply:
Thanks for the hint! Done :)